includes/Stories_Renderer/Renderer.php

```diff
@@ -413,10 +431,21 @@ protected function get_container_styles() {
 	public function render_single_story_content() {
 		$single_story_classes = $this->get_single_story_classes();
 		$show_story_player    = ( true !== $this->attributes['show_story_poster'] && $this->is_view_type( 'grid' ) );
+		$story_styles         = $this->is_view_type( 'circles' ) ? sprintf( '--size:%1$spx', $this->attributes['circle_size'] ) : '';
+
+		$lightbox_state          = "lightbox{$this->current()->get_id()}";
+		$lightbox_set_state_attr = ( $this->is_amp_request() ) ? sprintf(
+			'on="tap:AMP.setState({%1$s: ! %1$s})"',
+			$lightbox_state
+		) : '';
 
 		?>
 
-		<div class="<?php echo esc_attr( $single_story_classes ); ?>">
+		<div
+			class="<?php echo esc_attr( $single_story_classes ); ?>"
+			<?php echo wp_kses( $lightbox_set_state_attr, 'on' ); ?>
+			style="<?php echo esc_attr( $story_styles ); ?>"
+		>
```

Comment:
Using `$lightbox_set_state_attr` and `wp_kses` here doesn't seem very readable IMO.

What do you think about something like this?

```php
public function render_single_story_content() {
// ...

if ( $this->is_amp_request() ) {
?>
<div
	class="<?php echo esc_attr( $single_story_classes ); ?>"
	on="<?php echo esc_attr( sprintf( 'tap:AMP.setState({%1$s: ! %1$s})', $lightbox_state ) ); ?>"
	style="<?php echo esc_attr( $story_styles ); ?>"
>
<?php
	  if ( true === $show_story_player ) {
		  $this->render_story_with_story_player();
	  } else {
		  $this->render_story_with_poster();
	  }
?>
</div>
<?php
} else {
?>
<div
	class="<?php echo esc_attr( $single_story_classes ); ?>"
	style="<?php echo esc_attr( $story_styles ); ?>"
>
<?php
	  if ( true === $show_story_player ) {
		  $this->render_story_with_story_player();
	  } else {
		  $this->render_story_with_poster();
	  }
?>
</div>